modules/artifact/src/main/java/org/mule/module/artifact/classloader/ArtifactClassLoader.java

```diff
@@ -58,4 +58,9 @@
      */
     void addShutdownListener(ShutdownListener listener);
 
+    /**
+     * @return lookup policy used on the classLoader. Non null.
+     */
+    ClassLoaderLookupPolicy getClassLoaderLookupPolicy();
```

Comment:
Seems there are two or three hierarchies all with this method, anyway to clean this up? At least there are three definitions or implementations without override annotation.